gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -434,22 +436,17 @@ def create_network_function(self, context, network_function_info):
             return None
 
         if base_mode_support:
-            network_function_details = self.get_network_function_details(
-                network_function['id'])
-            network_function_data = {
-                'network_function_details': network_function_details
-            }
-            self.configurator_rpc.create_network_function_user_config(
-                network_function_data, service_config)
-
-            '''#(TODO) make changes for config_init and ansible too
-            template = service_config.split('heat_config:')[1]
-            network_function = {
-                'service_config': template
-            }
-            network_function = self.db_handler.update_network_function(
-                self.db_session, network_function_id, network_function)
-            '''
+            if tag_str != 'config_init:':
```

Comment:
What does base_mode_support indicate? This parsing applies to both base_mode and advance_mode.